app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx

```diff
@@ -168,6 +177,30 @@ export class Container extends React.Component<IContainerProps, IContainerState>
       return { label: `${msg} (${image.imageId})`, value: image.imageId };
     });
 
+    const dirtyTargetGroupList = dirtyTagetGroups ? dirtyTagetGroups.map(function (targetGroup, index){
+        return (
+          <li key={index}>{targetGroup}</li>
+        );
+      }) : '';
+
+    const dirtyTargetGroupSection = (
+       <div className="alert alert-warning">
+         <p>
+           <i className="fa fa-exclamation-triangle"></i>
+           The following target groups could not be found in the selected account/region/VPC and were removed:
+         </p>
+         <ul>
+           {dirtyTargetGroupList}
+         </ul>
+         <p className="text-right">
+           <a
+             className="btn btn-sm btn-default dirty-flag-dismiss"
+             onClick={() => this.updateDirtyTargetGroups()}
+           >Okay</a>
+         </p>
+       </div>
+       );
```

Comment:
What happens if I select another target group, do I still see the error? Do I have to manually dismiss it (ideally it will dismiss itself if the target group is no longer valid)?

Reply:
If you select another target group and saves the panel then the error won't appear again. But if you click **okay** on the error and saves the panel without choosing another target group then next time when you open the SG panel you will see the error.

Reply:
I think a user should continue to see the alert till they fix the underlying issue (as this is an invalid deployment).

Reply:
Do you mean we shouldn't be giving **okay** button to close the error?

Reply:
Yes. If there's a dirty target group, it means that the target group is no longer available to deploy, so the user NEEDS to update it or the deployment will fail. We shouldn't be giving them an option to clear the field since it's broken at that point.

Reply:
I got your point, I was just trying to be consistent with the AWS implementation of dirty target groups. For AWS provider as well **okay** button has been provided. Even when user clicks on the **okay** button dropdown value of the target group will not have the value of the dirty target groups.

Reply:
IIRC, the AWS provider doesn't save the state, it allows you to press okay, but when you reopen the window, it will show up again. That being said, I think this behaviour is better suited since it's a problem.

Reply:
In our case also if user doesn't choose the active target group then the error will re-appear once user re-opens the SG wizard.

Reply:
@piradeepk As per the latest commit, we are hiding the error message only when user chooses the active target group.